packages/health-gorilla-core/package.json

```diff
@@ -1,20 +1,9 @@
 {
-  "name": "@medplum/health-gorilla",
-  "version": "3.2.9",
+  "name": "@medplum/health-gorilla-core",
+  "version": "3.2.8",
+  "private": true,
   "description": "Medplum Health Gorilla SDK",
-  "homepage": "https://www.medplum.com/",
-  "bugs": {
-    "url": "https://github.com/medplum/medplum/issues"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/medplum/medplum.git",
-    "directory": "packages/health-gorilla"
-  },
-  "license": "Apache-2.0",
-  "author": "Medplum <hello@medplum.com>",
   "sideEffects": false,
-  "type": "module",
```

Comment:
We're trying to move all projects to `module` over time.  Any reason not to here?